ocrd/ocrd/server/models/job.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from typing import List, Optional
+
+from beanie import Document
+from pydantic import BaseModel
+
+
+class StateEnum(str, Enum):
+    queued = 'QUEUED'
+    running = 'RUNNING'
+    success = 'SUCCESS'
+    failed = 'FAILED'
+
+
+class JobInput(BaseModel):
+    path: str
+    description: Optional[str] = None
+    input_file_grps: List[str]
+    output_file_grps: Optional[List[str]]
```

Comment:
Why is `output_file_grp` optional but `input_file_grps` is not?

Reply:
Is there a case when a processor can run without input file groups?